### PR TITLE
Feature - Data containment levels

### PR DESCRIPTION
**Pull request by mihnea-radulescu**
Fixes the issue [#1979](https://github.com/flathub/website/issues/1979).

### CODE REVIEW

File: frontend/src/safety.ts [hunk 1]
```diff
@@ -550,11 +591,19 @@ function specificFileHandling(
       )
       if (partialMatch.length > 0 && fileSystem.partialMatchKey) {
         partialMatch.forEach((x) => {
+          const translationResult = readWriteTranslationKey(x)
+
+          highestDataContainmentLevel = Math.max(
```
Comment (razzeee) [inline frontend/src/safety.ts:596]: Might be nicer, to just do this at the end before line 617 based on the contents of `appSafetyRating`?

**Comment by razzeee** [comment]
I like this as an idea, but I would like to talk with the gnome software maintainers about this, would you be open to creating an issue on their bug tracker and reaching out?

File: frontend/src/safety.ts (hunk 1 above)
Comment (mihnea-radulescu) [reply frontend/src/safety.ts]: Thanks for the suggestion, implemented!

**Comment by mihnea-radulescu** [comment]
Thank you for the feedback, @razzeee!

I have created [an issue](https://gitlab.gnome.org/GNOME/gnome-software/-/issues/2340) on Gnome GitLab.

**Comment by mihnea-radulescu** [comment]
Closed due to a lack of interest from the GNOME community.

**Pull request closed without merging**